sparse_strips/vello_common/src/flatten.rs

```diff
@@ -111,10 +111,7 @@ pub fn fill(path: &BezPath, affine: Affine, line_buf: &mut Vec<Line>) {
 
 /// Flatten a stroked bezier path into line segments.
 pub fn stroke(path: &BezPath, style: &Stroke, affine: Affine, line_buf: &mut Vec<Line>) {
-    // TODO: Temporary hack to ensure that strokes are scaled properly by the transform.
-    let tolerance = TOL / affine.as_coeffs()[0].abs().max(affine.as_coeffs()[3].abs());
-
-    let expanded = kurbo::stroke(path.iter(), style, &StrokeOpts::default(), tolerance);
+    let expanded = kurbo::stroke(path.iter(), style, &StrokeOpts::default(), TOL);
```

Comment:
> The main reason I wanted to remove it is because it's causing trouble with transforms where the scaling factor is close to 0 and the skewing factor is instead pretty large. But maybe we need to come up with a more principled approach here.

@LaurenzV Would the issues still be fixed calculating the following?

```suggestion
    let tolerance = TOL / affine.as_coeffs()[0].abs().max(affine.as_coeffs()[3].abs()).max(1.);
    
    let expanded = kurbo::stroke(path.iter(), style, &StrokeOpts::default(), tolerance);
```

This doubles down on the current hack, ensuring the tolerance is always at least `TOL`, only allowing to become less tolerant for scaled-up geometry.

Reply:
As far as I can tell that does seem to work as well, so will adopt, thanks!